recipes/tensorflow-addons/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set version = "0.14.0" %}
+
+package:
+  name: tensorflow-addons
+  version: {{ version }}
+
+source:
+  url: https://github.com/tensorflow/addons/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: da9eaf683192054db73941e0de34d83754be4054dc92c409bc4693c3f3091586
+
+build:
+  number: 0
+  # missing dependency: https://github.com/conda-forge/tensorflow-feedstock/issues/124
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - bazel
+  host:
+    # GPU requirements for CUDA
+    - cudnn           # [cuda_compiler_version != "None"]
+    - python
+    - pip
+    - libprotobuf
+    # check tensorflow-range per addons-version, see
+    # https://github.com/tensorflow/addons#python-op-compatibility-matrix
+    - tensorflow 2.6.*
+    - tensorflow =*=cpu*   # [cuda_compiler_version == "None"]
+    - tensorflow =*=cuda*  # [cuda_compiler_version != "None"]
+    - typeguard >=2.7
+  run:
+    - python
+    - protobuf
+    # due to lack of ABI, we need the same version at runtime, see
+    # https://github.com/tensorflow/addons#c-custom-op-compatibility-matrix
+    - tensorflow 2.6.*
+    - tensorflow =*=cpu*   # [cuda_compiler_version == "None"]
+    - tensorflow =*=cuda*  # [cuda_compiler_version != "None"]
+
+test:
+  requires:
+    - pytest
+  imports:
+    - tensorflow_addons
+  commands:
+    - pytest -v
+
+about:
+  home: https://github.com/tensorflow/addons
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: Useful extra functionality for TensorFlow 2.x maintained by SIG-addons
+  dev_url: https://github.com/tensorflow/addons
+
+extra:
+  recipe-maintainers:
+    - h-vetinari
```

What about this simplification?
```suggestion
    - conda-forge/tensorflow
```

---

If that works, that's amazing!